ds-css/package.json

```diff
@@ -30,24 +30,28 @@
     "@babel/core": "^7.6.2",
     "@babel/plugin-transform-runtime": "^7.6.2",
     "@babel/preset-env": "^7.6.2",
-    "@fortawesome/fontawesome-free": "^5.11.2",
     "babel-loader": "^8.0.6",
     "bulma": "^0.7.5",
     "clean-webpack-plugin": "^3.0.0",
+    "copy-webpack-plugin": "^5.0.5",
     "css-loader": "^3.2.0",
+    "cssunminifier": "https://github.com/HIGGO36/cssunminifier/tarball/master",
```

Comment:
Let's find another way to do this. It may not be necessary for the ds style guide builds